crates/turbopack-core/src/issue/mod.rs

```diff
@@ -73,6 +73,42 @@ impl Display for IssueSeverity {
     }
 }
 
+#[derive(Clone, Debug)]
+#[turbo_tasks::value(shared)]
+pub enum StyledString {
+    Line(Vec<StyledString>),
+    String(String),
+    Pre(String),
```

Comment:
I would rename this to `Code`. `pre` is a bit html specific

Reply:
My concern was that the `Code` name wouldn't align with uses like styling file paths as monospaced.